src/components/audio-feedback.js

```diff
@@ -28,23 +30,51 @@ export function getAudioFeedbackScale(fromObject, toObject, minScale, maxScale,
   return Math.min(maxScale, minScale + (maxScale - minScale) * volume * 8 * distance);
 }
 
+function updateVolume(component) {
+  const newVolume = calculateVolume(component.analyser, component.levels);
+
+  component.loudest = Math.max(component.loudest, newVolume);
+  component.volume = component.loudest === 0 ? 0 : THREE.Math.mapLinear(newVolume, 0, component.loudest, 0, 1);
+
+  const s = 0.3;
+  component.volume = s * component.volume + (1 - s) * component.prevVolume;
+  component.loudest *= 0.999;
+
+  component.prevVolume = component.volume;
+}
+
+// function updateVolume(component) {
```

Comment:
remove

Reply:
Yep, just wanted to show a different method I tried as well, and maybe give you a chance to play with it.